assets/app/view/game/buy_sell_shares.rb

```diff
@@ -136,7 +136,7 @@ def render_exchanges
 
         @game.companies.each do |company|
           @game.abilities(company, :exchange) do |ability|
-            next if ability.corporation != @corporation.name && ability.corporation != 'any'
+            next unless ability.corporations(@game).include?(@corporation)
```

Comment:
Just a thought, would it make more sense to be @game.exchange_corporations(abillity)? That seems to be the general pattern.

Reply:
yes it should be game.... please change and we can merge

Reply:
Fixed